python/ray/util/client/options.py

```diff
@@ -59,15 +14,11 @@ def validate_options(kwargs_dict: Optional[Dict[str, Any]]) -> Optional[Dict[str
 
     out = {}
     for k, v in kwargs_dict.items():
-        if k not in options.keys():
-            raise TypeError(f"Invalid option passed to remote(): {k}")
-        validator = options[k]
-        if len(validator) != 0:
-            if v is not None:
-                if not isinstance(v, validator[0]):
-                    raise ValueError(validator[2])
-                if not validator[1](v):
-                    raise ValueError(validator[2])
+        if k not in options.valid_options:
```

Comment:
Should we validate task and actor options separately since they have their own unique set of options?

Reply:
That's for the future, I do not want to further increase the LoC, otherwise it takes much longer time to merge.